### PR TITLE
PXP-417: [Wukong CLI] Don't display selection if it's not available for an application

### PR DESCRIPTION
**Pull request by jk-gan**
## Summary

The namespace selections always show `Prod` and `Staging`, and the version selections always show `Blue` and `Green` even if the current application doesn't have that namespace/version. Now the selections will be based on the API result.

If the user is using `--namespace` or `--version` flag but the value provided is not available for the current application, the cli will return `DeploymentError`.
 

Ticket: [PXP-417](https://mindvalley.atlassian.net/browse/PXP-417)

### Added
- [x] Add `DeploymentError`
Screenshot 2022-11-09 at 2 40 17 PM

### Changed
- [x] The namespace and version selections now depend on the API result during `wukong deployment execute`
Screenshot 2022-11-09 at 2 42 05 PM
Screenshot 2022-11-09 at 2 42 08 PM

### CODE REVIEW

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/wukong-cli/pull/40#" title="Author self-approved">jk-gan</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fwukong-cli).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/wukong-cli/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by onimsha — APPROVED** [review]

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/wukong-cli/pull/40#" title="Author self-approved">jk-gan</a>*, *<a href="https://github.com/mindvalley/wukong-cli/pull/40#pullrequestreview-1175030402" title="Approved">onimsha</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fwukong-cli).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/wukong-cli/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
